src/extensions/components/registration/agencySelection/AgencySelectionResults.tsx

```diff
@@ -50,11 +51,31 @@ export const AgencySelectionResults = ({
 	const { setDisabledNextButton, registrationData } =
 		useContext(RegistrationContext);
 	const { consultant: preselectedConsultant } = useContext(UrlParamsContext);
-
 	const [selectedAgency, setSelectedAgency] = useState<AgencyDataInterface>(
 		registrationData?.agency
 	);
 
+	const onlyExternalAgencies = results?.every((agency) => agency.external);
+	const isSingleResultAndNotOnlyExternal =
+		results?.length === 1 && !onlyExternalAgencies;
+
+	useEffect(() => {
+		if (!isSingleResultAndNotOnlyExternal) return;
+
+		const handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
1. Could you verify if other stuff is still doing nothing? For example if you have focus on back button
2. onNextClick doesn't have the logic to check if disabledNextButton is false. So it needs to be checked here.

Reply:
I removed the Enter listener for now